### PR TITLE
RBMHUIM-2500: JS find user integration example

### PR DESCRIPTION
**Pull request by PatrickSchuster**
JS integration example for finding a user using the API.

### CODE REVIEW

File: js/src/examples/find-user/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Find a user
+Finds a user using the Client API.
+
+File: `find-user.js`
+
+REST documentation: https://uim.redbull.com/uim/resources/docs/rest-api.html#client-resources-form-submissions-create
```
Comment (pablotp) [inline js/src/examples/find-user/README.md]: This is pointing to "Form Submissions Crete". The correct one should be: https://uim.redbull.com/uim/resources/docs/rest-api.html#client-resources-user-get-user

File: js/src/examples/find-user/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+### 2. Run the docker image
+`docker run --rm --name find-user -e "APPLICATION_ID=<your_app_id>" -e "API_KEY=<your_api_key>" -e "ENVIRONMENT=design" <your username>/node-web-app`
+ 
+## Optional: Execute the script manually (instead of running the docker image) 
+`node src/find-user.js`
+
+## <a name="variables">Variables</a>
+
+The variables you would need to run this example against the `design` environment are:
+* `Application_ID`
```
Comment (pablotp) [inline js/src/examples/find-user/README.md]: If we are case-sensitive is should be `APPLICATION_ID`, right?

File: js/src/examples/find-user/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "js_form_submission",
```
Comment (pablotp) [inline js/src/examples/find-user/package.json]: `js_find_user` ❓

File: js/src/examples/find-user/package.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "1.0.0",
+  "description": "Form submission in JS",
+  "author": "patrick.schuster@e-mundo.de",
+  "main": "src/form-submission.js",
+  "scripts": {
+    "start": "node src/form-submission.js"
```
Comment (pablotp) [inline js/src/examples/find-user/package.json]: idem

**Comment by mteufner** [comment]
This does not apply to this PR specifically, but to our JS integration examples in general:
Why do we have a separate readme for every JS example, but only one accumulated readme for Java/PHP? Just a minor detail, but not very consistent.

**Comment by PatrickSchuster** [comment]
> This does not apply to this PR specifically, but to our JS integration examples in general:
> Why do we have a separate readme for every JS example, but only one accumulated readme for Java/PHP? Just a minor detail, but not very consistent.

I used separate READMEs for each JS example since they are independent projects. This was done after a comment on my last JS integration example: https://github.com/redbullmediahouse/uim-integration-examples/pull/8#discussion_r395560758

File: js/src/examples/find-user/README.md (hunk 2 above)
Comment (PatrickSchuster) [reply js/src/examples/find-user/README.md]: You are right!

File: js/src/examples/find-user/README.md (hunk 1 above)
Comment (PatrickSchuster) [reply js/src/examples/find-user/README.md]: Thanks!

**Comment by pablotp** [comment]
> > This does not apply to this PR specifically, but to our JS integration examples in general:
> > Why do we have a separate readme for every JS example, but only one accumulated readme for Java/PHP? Just a minor detail, but not very consistent.
> 
> I used separate READMEs for each JS example since they are independent projects. This was done after a comment on my last JS integration example: [#8 (comment)](https://github.com/redbullmediahouse/uim-integration-examples/pull/8#discussion_r395560758)

@mteufner in the JS examples it is a bit different as there isn't a main docker image from which you execute all examples but each example has it's own image.
@gpotzkov suggested to have a selector (like in Java), but it hasn't been investigated how this could be done for the JS examples (in Java it is done via gradle).

**Pull request merged**